f8a_worker/storages/postgres.py

```diff
@@ -213,7 +213,7 @@ def store_in_bucket(content):
         s3 = StoragePool.get_connected_storage('S3UserProfileStore')
         s3.store_in_bucket(content)
 
-    def store_api_requests(self, external_request_id, data, dep_data):
+    def store_api_requests(self, external_request_id, uuid_data, data, dep_data):
```

Comment:
There is scope for clean up in this table, a lot of the information that we store is not useful. A lot of data is none and I am not sure why those columns exist. It will be good to store the ecosystem that is used in plain text. I also believe that it useful to store the stack itself.

cc @arajkumar @yzainee